modules/RTC/TraceablePeerConnection.js

```diff
@@ -709,7 +710,11 @@ TraceablePeerConnection.prototype._remoteTrackAdded = function(stream, track) {
     }
 
     const muted = peerMediaInfo.muted;
-    const videoType = peerMediaInfo.videoType; // can be undefined
+
+    // peerMediaInfo.videoType can be undefined so set a default type in case
+    // there is logic depending on type existing and rely on a presence update
+    // to come in to update to the correct type.
+    const videoType = peerMediaInfo.videoType || VideoType.CAMERA;
```

Comment:
so why default it to camera here instead of have logic that checks the type handle the case where it is `undefined`?

Reply:
Honestly, I couldn't decide which to do; either set a default when creating the remote jitsi track or handle undefined as needed. I couldn't tell for sure what places would break if videoType were undefined so this seemed like a good enough place to handle the undefined. Do you have a preference for handling errors due to undefined as they show up?

Reply:
after we talked about it earlier, i think keeping this as undefined and making other code handle undefined is better in the long run.

if we find that to be too problematic, then we know this event needs to wait until we know what the videoType is (or those places need to listen for another event).

i don't like marking it as camera because we are explicitly guessing at information we don't actually have/know.